### ISSUE TITLE
28695377: iMessage Apps: No option to remove MSMessage via VoiceOver

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#16089]
#### Description

The "x" Close button in a MSMessage to remove it as a draft is not accessible via VoiceOver.
## 

Product Version: 10.1 beta 2
Created: 2016-10-10T16:18:44.360670
Originated: 2016-10-10T18:18:00
Open Radar Link: http://www.openradar.me/28695377
